sparkle/src/components/Button.tsx

```diff
@@ -158,23 +167,31 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     );
 
     const buttonElement = (
-      <MetaButton
-        ref={ref}
-        size={buttonSize}
-        variant={variant}
-        disabled={isLoading || props.disabled}
-        className={isPulsing ? "s-animate-pulse" : ""}
-        aria-label={ariaLabel || tooltip || label}
-        style={
-          {
-            "--pulse-color": "#93C5FD",
-            "--duration": "1.5s",
-          } as React.CSSProperties
-        }
-        {...props}
+      <LinkWrapper
+        href={href}
+        target={target}
+        rel={rel}
+        replace={replace}
+        shallow={shallow}
       >
-        {content}
-      </MetaButton>
+        <MetaButton
```

Comment:
Wait I thought that the button was support href, directly?

Reply:
Not that I know of? See [docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLButtonElement)

Reply:
I meant our own `Button` component.

Reply:
Nop it unfortunately doesn't and I'm trying to bring it here. Tried to follow the shadcn pattern -> wrapping the link inside a button but it then breaks the Tabs. Any suggestion ?